utils/params_utils.py

```diff
@@ -77,8 +79,8 @@ def create_hparams(args, target_vocab_size=None, binf_count=None, sos_id=1, eos_
         is_reset = args.reset
     except AttributeError:
         is_reset = False
-    if os.path.exists(hparams_file) and not is_reset:
-        with open(hparams_file, 'r') as f:
+    if tf.gfile.Exists(hparams_file) and not is_reset:
+        with tf.gfile.Open(hparams_file, 'r') as f:
```

Comment:
This function is deprecated, let's switch to newer API: `tf.io.gfile.GFile()`

Reply:
There is no mention about deprecation in the documentation.

Reply:
TF 1.14.0:
`W0806 16:56:12.094715 139758041270080 deprecation_wrapper.py:119] From ...: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.`